crates/aptos-debugger/Cargo.toml

```diff
@@ -14,6 +14,7 @@ rust-version = { workspace = true }
 
 [dependencies]
 anyhow = { workspace = true }
+aptos-consensus = { workspace = true }
```

Comment:
Such dependencies make our tools too heavy. I think we should consider move `aptos-consensus-types` to a separate crate. (May be in a separate PR but I don't know where we actually track these things).

Reply:
aptos-consensus-types is already a separate crate..

There are some other things need to be moved to separate crates to avoid this dependency, e.g. consensus/quorum db.. I will leave it for future PR.